pybamm/models/base_models.py

```diff
@@ -183,6 +186,15 @@ def jacobian(self):
     def jacobian(self, jacobian):
         self._jacobian = jacobian
 
+    @property
+    def use_capacitance(self):
+        return self._use_capacitance
+
+    @use_capacitance.setter
+    def use_capacitance(self, use_capacitance):
```

Comment:
where is `use_capacitance` used? I'm guessing that it governs the equations that are put in the model (during __init__). So would calls to this setter after the model is created have any effect? Should this be a read-only option?

Reply:
Yes, will change this to read-only